src/Adapter/Json.php

```diff
@@ -45,6 +45,7 @@ public function getOptions()
         if ($this->options === null) {
             $this->options = new JsonOptions();
         }
+
```

Comment:
I can revert.  I thought as I was looking at the file, I add the space for consistency.